README.md

```diff
@@ -1,10 +1,50 @@
-# dd-trace-go
-
 [![CircleCI](https://circleci.com/gh/DataDog/dd-trace-go/tree/master.svg?style=svg)](https://circleci.com/gh/DataDog/dd-trace-go/tree/master)
 [![Godoc](http://img.shields.io/badge/godoc-reference-blue.svg?style=flat)](https://godoc.org/github.com/DataDog/dd-trace-go/tracer)
 
-A Go tracing package. Currently requires at least Go 1.7.
+A Go tracing package for Datadog APM.
+
+## Contributing Quick Start
+
+Requirements:
+
+* Go 1.7 or later
+* Docker
+* Rake
+* gometalinter – `go get -u github.com/alecthomas/gometalinter`
+* golint – `go get -u github.com/golang/lint/golint`
+* goconst – `go get -u github.com/jgautheron/goconst/cmd/goconst`
+* gas – `go get -u github.com/GoASTScanner/gas`
+* ineffassign – `go get -u github.com/gordonklaus/ineffassign`
+* gocyclo – `go get -u github.com/fzipp/gocyclo`
+
+### Run the tests
+
+Start the containers defined in `docker-compose.yml`:
+
+```
+$ docker-compose up -d
+```
+
+Fetch this package's dependencies:
+
+```
+$ rake get
```

Comment:
More a question for the owners of this repo:

Should we start locking dependency versions using Glock or Glide?

Reply:
At the moment we only have one dependency that is `msgpack` and it's vendored so no need to add a vendoring tool here. The only reason why we're providing a `rake get`, is because we need those "virtual dependencies" in our tests, but not in our library. Indeed, `dd-trace-go` doesn't require anything to be used.

This comment should be addressed in a different way that I'll explain in the comments feed. About glide / dep, we may consider to use `dep` as a default one since it has been elected as the language package manager that will be included in Go itself (at some point).